packages/adapter-sveltekit/src/hooks/documentation-read.ts

```diff
@@ -72,17 +74,35 @@ export const documentationRead: DocumentationReadHook<PluginOptions> = async (
 			`;
 		}
 
-		let componentFileContent = source`
-			<script>
-				import { SliceZone } from "@prismicio/svelte";
+		let componentFileContent;
 
-				import { components } from "$lib/slices";
+		if (isSvelte5) {
+			// Svelte 5 syntax with runes
+			componentFileContent = source`
+				<script>
+					import { SliceZone } from "@prismicio/svelte";
 
-				export let data;
-			</script>
+					import { components } from "$lib/slices";
 
-			<SliceZone slices={data.page.data.slices} {components} />
-		`;
+					let { data } = $props();
+				</script>
+
+				<SliceZone slices={data.page.data.slices} components={components} />
```

Comment:
This shortcut should still work in Svelte 5:

```suggestion
				<SliceZone slices={data.page.data.slices} {components} />
```